### DISCUSSION TITLE
Simple timer mode

### DISCUSSION THREAD

**Discussion by ghost** [Hanziness/AnotherPomodoro#183] [Ideas & feature requests]
Although this is a bit beyond the scope of the application, but it would be cool to write the time and turn on a simple timer.

**Comment by Hanziness** [comment]
I have been thinking of creating a "programmable" way to set up schedules, which would make the app flexible enough for this functionality, but I'm afraid it's a bit farther into the future. Thank you for the suggestion, though, I'll keep it in mind if I happen to include a feature that would allow this.
